string Sender
uint32 JointId
---
string JointName
uint32 JointType